woocommerce/woocommerce-gutenberg-products-block
# Remove minConstraint and maxConstraint from price filter state

`minConstraint` and `maxConstraint` could be directly inferred from `results`, so there was no need to save them in the state.

### How to test the changes in this Pull Request:

1. Create a post with a _Price Filter_ block, an _Attributes Filter_ block and an _All Products_ block.
2. Preview the post.
3. Modify the filtered attributes and verify the _Price Filter_ min and max values update accordingly (there are no regressions).
